expected/16.0/extra/float4.out

```diff
@@ -1260,7 +1260,601 @@ begin
   end loop;
 end;
 $d$;
---Testcase 183:
+--Testcase 280:
+CREATE FOREIGN TABLE "type_FLOAT_INF"( "i" int OPTIONS (key 'true'), "f" float8) SERVER sqlite_svr OPTIONS (table 'type_FLOAT_INF');
+--Testcase 281:
+CREATE FOREIGN TABLE "type_FLOAT_INF+"( "i" int OPTIONS (key 'true'), "f" float8, "t" text, "l" smallint) SERVER sqlite_svr OPTIONS (table 'type_FLOAT_INF+');
+--Testcase 282:
+SELECT * FROM "type_FLOAT_INF+";
+ i |     f     |  t   | l 
+---+-----------+------+---
+ 1 | -Infinity | real | 4
+ 2 |  Infinity | real | 3
+ 3 | -Infinity | real | 4
+ 4 |  Infinity | real | 3
+ 5 |   -1e+308 | real | 9
+ 6 |         0 | real | 3
+ 7 |    1e+308 | real | 8
+(7 rows)
+
+--Testcase 283:
+INSERT INTO "type_FLOAT_INF" (i, f) VALUES (10, 'Infinity');
+--Testcase 284:
+INSERT INTO "type_FLOAT_INF" (i, f) VALUES (11, '+Infinity');
+--Testcase 285:
+INSERT INTO "type_FLOAT_INF" (i, f) VALUES (12, '-Infinity');
+--Testcase 286:
+INSERT INTO "type_FLOAT_INF" (i, f) VALUES (13, 'Inf');
+--Testcase 287:
+INSERT INTO "type_FLOAT_INF" (i, f) VALUES (14, '+Inf');
+--Testcase 288:
+INSERT INTO "type_FLOAT_INF" (i, f) VALUES (15, '-Inf');
+--Testcase 289: Inserted as NULL!!!
+--see https://github.com/sqlite/sqlite/blob/6db0b11e078f4b651f0cf00f845f3d77700c1a3a/src/vdbemem.c#L973
+--INSERT INTO "type_FLOAT_INF" (i, f) VALUES (16, 'NaN');
```

Comment:
Please remove this testcases and related comment

Reply:
Done

Reply:
Thank you for supporting. It's OK